docs/main/05-reference/04-promises/03-updates.md

```diff
@@ -4,26 +4,56 @@ sidebar_label: Updates
 description: Documentation on how updates behave for Promises
 ---
 
-# Updates
+Kratix supports updating Promises with new specifications.
 
-Kratix supports updating Promises with new specifications. Any update to the
-Promise will result in the re-running the Configure Workflows for all resources.
-For example, if you update the Configure Workflow image version and change
-a field in the Dependencies Kratix will roll out the new Dependencies to
-Destinations and re-run all the Configure Workflows
-with the new image version.
+An update to a Promise will cause Kratix to reconcile on the new Promise definition,
+and any changes will be rolled out during this reconciliation.
+
+This may include:
+- Updating the Promise API, which rolls out an update to the underlying CRD for the
+  Resources managed by the Promise.
+- Updating the Promise or Resource workflows.
+- Updating the Promise scheduling.
+- Updating the Promise's static dependencies (the `dependencies` field in the Promise
+  spec).
```

Comment:
nothing against this addition, but is it adding any extra clarity (i.e. is "a promise update" something confusing that require examples?)

Reply:
We've just come back to this and think it's probably worth keeping, if only to spell out to Kratix users what they can update